### ISSUE TITLE
Rotation

### ISSUE THREAD

**Issue by SelenaClover22** [cszyzhang/RISurConv#4]
Thanks for the great work!

I have one question when trying to replicate your results: in the paper, it shows that the approach demonstrates great stability for rotations, but when I'm checking the code(e.g. for ModelNet), there does not seem to be rotation procedures on the training or test data, and the preprocessing is just for sampling and normalization. I did see some rotation functions in provider.py, but I did not find where you used them to train the e.g. classification task.

**Comment by SelenaClover22** [comment]
and one more issue: PyTorch 1.9.0 is not compatible with CUDA 12.1, I wonder if another version of PyTorch or CUDA was used..
